website/docs/cloud-docs/workspaces/variables/index.mdx

```diff
@@ -152,3 +152,15 @@ When you trigger a run through the command line, Terraform Cloud applies the fol
 - **C_Variable_Set:** `KEY2`. For this run, `KEY2` equals `c`. This is a global variable set with no overwritten values.
 
 ![An example scenario demonstrating variable precedence in Terraform Cloud](/img/docs/variable-precedence-example.png)
+
+
+## Precedence with Priority Variable Sets
+
+You can select to prioritize all values of the variables in a variable set.
+When a variable set is priority, the values take precedence over any variables with the same key set at a more specific scope.
+
+For example, variables in a priority global variable set would take precedence over any variables with the same key in a priority project or workspace scoped variable set. Variables in a priority variable set will take precedence over any variables with the same key set through the CLI on a run.
+
+If two priority variable sets with the same scope include the same variable key, Terraform Cloud will determine precedence by the alphabetical order of the variable sets' names.
+
+While a priority variable set can enforce that terraform variables use designated values, it does not guarantee that the variable is used in configuration. It is still possible for a user to directly modify the Terraform configuration and remove usage of a variable and replace it with a hard coded value. If you are looking for stricter enforcement than this, we recommend using policy checks or run tasks.
```

Comment:
```suggestion
If you mark a variable set as `priority`, all of its values take precedence over any other variables with the same key, even if they have a more specific scope. For example, variables in a priority global variable set take precedence over same-key variables in a priority project- or workspace-scoped variable set. Refer to the [precedence](#precedence) section above for the complete precedence hierarchy.

If two priority variable sets with the same scope include the same variable key, Terraform Cloud will determine precedence by the alphabetical order of the variable sets' names.

While a priority variable set can enforce specific variable values, it does not guarantee that the configuration actually references that variable instead of hardcoding a value. For stricter enforcement mechanisms, we recommend using policy checks or run tasks.
```